nginx/entrypoint.sh

```diff
@@ -15,6 +15,7 @@ CERT_PUBLIC_PATH=${CERT_PUBLIC_PATH-"/certs/fullchain.pem"}
 CERT_PRIVATE_PATH=${CERT_PRIVATE_PATH-"/certs/privkey.pem"}
 HTTPS_PORT=${HTTPS_PORT-"443"}
 TLS_PROTOCOLS=${TLS_PROTOCOLS-"TLSv1 TLSv1.1 TLSv1.2"}
+SERVER_NAMES_HASH_SIZE=${SERVER_NAMES_HASH_SIZE-"32"}
```

Comment:
I think there should be a longer default, at least 64. Probably 128.

Did you find any reasons against it?

Reply:
The value of `32` is the default value used by nginx, so i try to comply with existing code as much as possible.